src/main/kotlin/no/nav/bidrag/bidragskalkulator/dto/BeregningRequestDto.kt

```diff
@@ -44,5 +44,26 @@ data class BeregningRequestDto(
     @field:NotEmpty(message = "Liste over barn kan ikke være tom")
     @field:Valid
     @Schema(description = "Liste over barn som inngår i beregningen", required = true)
-    val barn: List<BarnDto>
+    val barn: List<BarnDto>,
+
+    @Schema(description = "Boforhold for den påloggede personen. Må være satt hvis bidragstype for barnet er PLIKTIG", required = false)
+    val dinBoforhold: BoforholdDto? = null,
```

Comment:
Tror det er "et boforhold", og derfor 
```suggestion
    val dittBoforhold: BoforholdDto? = null,
```